qabelbox/src/main/java/de/qabel/desktop/repository/sqlite/SqliteContactRepository.java

```diff
@@ -195,4 +204,71 @@ public synchronized Contact findByKeyId(Identity identity, String keyId) throws
             throw new EntityNotFoundExcepion("exception while searching contact: " + e.getMessage(), e);
         }
     }
+
+    @Override
+    public Collection<Pair<Contact, List<Identity>>> find(Identities identities, String searchString) throws PersistenceException {
```

Comment:
Couldn't this be implemented by just looping for the identities, calling find(identity) for them all and just aggregating this?